.github/workflows/bless.yml

```
@@ -0,0 +1,121 @@
+        id: nginx
+      
+      - name: Copy static test files
+        run: cp -r tests/fixtures/pmtiles2/* ${{ steps.nginx.outputs.html-dir }}
+      
+      - name: Initialize database
+        run: tests/fixtures/initdb.sh
+        env:
+          DATABASE_URL: ${{ steps.postgres.outputs.connection-uri }}
+      
+      - name: Install system dependencies
+        run: sudo apt-get install -y gdal-bin sqlite3-tools
+      
+      - name: Debug environment
+        run: |
+          echo "User: $USER"
+          echo "GitHub Actor: ${{ github.actor }}"
+          echo "Database URL: ${{ steps.postgres.outputs.connection-uri }}"
+      
+      - name: Run just bless
+        run: |
+          echo "Running just bless..."
+          just bless
+        env:
+          DATABASE_URL: ${{ steps.postgres.outputs.connection-uri }}
+      
+      - name: Commit blessed output
+        id: commit
+        run: |
+          git config --local user.email "action@github.com"
+          git config --local user.name "GitHub Action (bless)"
+          
+          echo "Checking for changes..."
+          git status
+          git add -A
+          
+          if ! git diff --cached --quiet; then
+            echo "Changes detected, committing..."
+            git commit -m "Bless test output [skip ci]"
+            echo "Pushing changes..."
+            git push origin ${{ github.event.pull_request.head.ref }}
+            echo "Test output blessed and committed"
+            echo "changes_made=true" >> $GITHUB_OUTPUT
+          else
+            echo "ℹ️ No changes to commit"
+            echo "changes_made=false" >> $GITHUB_OUTPUT
+          fi
+      
+      - name: Remove bless label
+        uses: actions/github-script@f28e40c7f34bde8b3046d885e986cb6290c5673b # v7.1.0
+        with:
+          script: |
+            try {
+              await github.rest.issues.removeLabel({
+                owner: context.repo.owner,
+                repo: context.repo.repo,
+                issue_number: context.issue.number,
+                name: 'bless'
+              });
+              console.log('Label removed successfully');
+            } catch (error) {
+              console.log('Could not remove label:', error.message);
+            }
+      
+      - name: Add comment
+        uses: actions/github-script@f28e40c7f34bde8b3046d885e986cb6290c5673b # v7.1.0
+        with:
+          script: |            
+            let body;
+            if ('${{ steps.commit.outputs.changes_made }}' === 'true') {
+              body = ;
+            } else {
+              body = 'ℹ️ No changes were needed. The test output already matches the expected results.';
+            }
+            
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: context.issue.number,
+              body: body
+            });
```

we can do a slight optimisation 😉

```suggestion
      - uses: autofix-ci/action@635ffb0c9798bd160680f18fd73371e355b85f27 # v1.3.2
```